default/stringtables/en.txt

```diff
@@ -5077,34 +5077,34 @@ BEGINNER_HINT_04
 Systems that are connected by thick lines are [[encyclopedia SUPPLY_TITLE]] line linked and they share [[encyclopedia INDUSTRY_TITLE]] Production Points (PP).
 
 BEGINNER_HINT_05
-All system share [[encyclopedia RESEARCH_TITLE]] Points (RP) that they generate even if they are not connected by supply lines. If you have a remote system that is unconnected, setting the [[encyclopedia FOCUS_TITLE]] to research might be a more efficient use.
+All system share [[encyclopedia RESEARCH_TITLE]] Points (RP) that they generate even if they are not connected by supply lines.  When a system is not connected, setting the [[encyclopedia FOCUS_TITLE]] to research might be more efficient than other settings.
 
 BEGINNER_HINT_06
-Ships will refuel when they are in a system that is connected by thick or by thin lines, as long as the thin lines extend back to thick supply line connected systems.
+Ships will refuel when they are in a system that is connected to a [[encyclopedia SUPPLY_TITLE]] line.  Narrow supply lines, which are displayed with a thin line between systems, will also refuel ships if they are connected to a wider supply line.
 
 BEGINNER_HINT_07
-Some buildings, like the [[buildingtype BLD_INDUSTRY_CENTER]], will boost production on all supply connected planets. Others, like [[buildingtype BLD_GAS_GIANT_GEN]], only help in the system that they are built in. Always check the 'Pedia for specifics on different buildings!
+Some buildings, like the [[buildingtype BLD_INDUSTRY_CENTER]], will have an effect on each planet that has a supply line connection to it. Others, like [[buildingtype BLD_GAS_GIANT_GEN]], only affect the system that they are built in.  Check the 'Pedia to learn about the functions of different buildings.
 
 BEGINNER_HINT_08
-An Outpost ship can setup an Outpost on any unclaimed visible planet or asteroid that it can reach. Outposts can be settled by any species in your empire that has a connection to it, at least 5 happiness, and 3 population.
+An Outpost ship can setup an outpost on any unclaimed visible planet or asteroid that it can reach, unless there are hostile ships in that system.
 
 BEGINNER_HINT_09
-Colony ships take longer to build than outpost ships but can settle planets outside of your supply range.
+Colony ships take longer to build than outpost ships, but can settle planets outside of your supply range.
```

Comment:
http://www.grammarly.com/handbook/punctuation/comma/23/comma-before-but/

```Colony ships take longer to build than outpost ships but can settle planets outside of your supply range.``` --> ok
```Colony ships take longer to build than outpost ships, but they can settle planets outside of your supply range.``` --> ok
```Colony ships take longer to build than outpost ships, but can settle planets outside of your supply range.``` --> not ok.